public/scripts/map.js

```diff
@@ -0,0 +1,154 @@
+(function(module) {
+
+mapCoordinates ={}
+var map = L.map('map').setView([47.6204,-122.3491], 13);
+L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png',
+  {
+    attribution: 'Tiles by <a href="https://www.openstreetmap.org/">MAPC</a>, Data by <a href="http://mass.gov/mgis">MassGIS</a>',
+    maxZoom: 17,
+    minZoom: 9
+  }).addTo(map);
+var pointsOnMap = new L.FeatureGroup();
+mapCoordinates.addPlacesToMap = function(){
+  Place.all.forEach(function(a){
+    pointsOnMap.clearLayers();
+    filterer.filteredResults.forEach(function(a){
+
+      if (a["lat-long"]) {
```

Comment:
This big block of `if / else if /else` functionality could be abstracted heavily and simplified.